src/styles/blocks/wrapper.scss

```
@@ -0,0 +1,6 @@
+.wrapper {
+  max-width: $maxWidthContent;
+  height: 100%;
+  margin: 0 auto;
```

```suggestion
  margin: 0 auto;
```
No outer spacings for BEM-blocks